theseus_gui/src/pages/Index.vue

```diff
@@ -1,17 +1,57 @@
 <script setup>
+import { ref, onMounted } from 'vue'
+import { ofetch } from 'ofetch'
+import { AnimatedLogo } from 'omorphia'
 import RowDisplay from '@/components/RowDisplay.vue'
 import { shallowRef } from 'vue'
 import { list } from '@/helpers/profile.js'
 
+const loading = ref(false)
+
+const featuredModpacks = ref({})
+const featuredMods = ref({})
+
 const profiles = await list()
 const recentInstances = shallowRef(Object.values(profiles))
+
+const getFeaturedModpacks = async () => {
+  const response = await ofetch(
+    'https://api.modrinth.com/v2/search?facets=[["project_type:modpack"]]&limit=10&index=follows'
+  )
+  featuredModpacks.value = response.hits
+}
+const getFeaturedMods = async () => {
+  const response = await ofetch(
+    'https://api.modrinth.com/v2/search?facets=[["project_type:mod"]]&limit=10&index=follows'
+  )
+  featuredMods.value = response.hits
+}
+
+const load = () => (loading.value = true)
+
+onMounted(async () => {
+  loading.value = true
+  await Promise.all([getFeaturedModpacks(), getFeaturedMods()])
+  loading.value = false
+})
```

Comment:
this should use script setup and suspense

Reply:
The loading animation wouldn't appear without using onMounted. Since that will be going away, will tear this out.